etc/ci/upload_nightly.sh

```diff
@@ -12,7 +12,12 @@ usage() {
 
 
 upload() {
-    s3cmd put "${2}" "s3://servo-builds/nightly/${1}/"
+    local s3path filename
+    s3path="s3://servo-builds/nightly"
+    filename=test=$(basename "${2}")
```

Comment:
Let's call this variable `package_filename`. The `test=` part should be removed. Put the command substitution in quotes.